server/models/GameLogic.js

```diff
@@ -262,6 +278,13 @@ class GameLogic {
         this.shufflePlayedCards();
         socket.emitPlayedCards(this.roomId, this.getPlayedCards());
         socket.promptPlayersVote(this.roomId, this.currentPlayer, voteDuration);
+        this.players.forEach(player => {
```

Comment:
Can this for loop be moved into it's own function like like play card for ais or something?

Reply:
And anywhere else that has a similar loop